docker-compose.yml

```diff
@@ -14,6 +14,9 @@ services:
       interval: 5s
       timeout: 3s
       retries: 5
+    networks:
+      chain_net:
+        ipv4_address: 172.27.0.106
```

Comment:
Is changing to ip addresses needed for postgres, childchain and watchers?

Reply:
@unnawut Initially, I added ip for the Geth node so can define Nginx config. But after adding Nginx and Geth to the same network, they became unavailable for other containers. That's why I added all containers to the same network

Reply:
Thanks for the clarification!